packages/leancode_hooks/lib/src/use_decarative_text_editing_controller.dart

```diff
@@ -1,6 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_hooks/flutter_hooks.dart';
 
+/// A wrapper around [useTextEditingController] that updates
+/// [TextEditingController.text] when it changes.
+///
+/// Does not handle some edge cases with the current cursor/selection.
```

Comment:
Could you explain a bit more about what cases are those and what happens then?

Reply:
@shilangyu 😬

Reply:
TBH I would remove this hook, it was used for a very specific usecase in our project

Reply:
I'd do this in `v0.0.4`